app/controllers/internal/profile_fields_controller.rb

```diff
@@ -1,32 +1,50 @@
 module Internal
   class ProfileFieldsController < Internal::ApplicationController
+    ALLOWED_PARAMS = %i[
+      input_type label active placeholder_text description
```

Comment:
**comment (blocking):** We merged the PR that adds `group` already, so I think we should add that here too.

```suggestion
      input_type label active placeholder_text description group
```

Additional suggestion:

When things are still in flux, I often use something like this:

```ruby
ALLOWED_PARAMS = (ProfileField.column_names - %w[id created_at updated_at]).freeze
```
you can always changed it to the explicit list in the final PR.

Reply:
Thanks @citizen428, correct - I'm leaving everything group related for the next PR where I tackle that. 

Also, thats a pretty cool tip thank you :)